doc/Overview/Configuration/ConfigurationOverview.md

```diff
@@ -1,141 +1,158 @@
 ---
 uid: Overview.Configuration
```

Comment:
Sequence of this doc seems a bit weird. I'd suggest changing to:
- UseConfiguration - makes IConfiguration available
- Embedded/Content sources
- Sections - this is how you access content
- IWriteable - this is how you can write settings